<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQCaptcha</class>
 <widget class="QDialog" name="QQCaptcha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_captcha_">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Captcha:</string>
         </property>
         <property name="buddy">
          <cstring>le_captcha_</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_captcha_"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pb_confirm_">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_confirm_</sender>
   <signal>clicked()</signal>
   <receiver>QQCaptcha</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
